openmc/material.py

```diff
@@ -311,6 +327,65 @@ def remove_nuclide(self, nuclide):
         if nuclide._name in self._nuclides:
             del self._nuclides[nuclide._name]
 
+    def add_macroscopic(self, macroscopic):
```

Comment:
I didnt do it because I thought maybe making this behave just like the Nuclide class would be the most straight-forward way for the user.  I admit that your suggestion definitely makes a cleaner code base though.